pymeasure/instruments/racal/racal1992.py

```diff
@@ -0,0 +1,363 @@
+            map_values=True
+        )
+
+    delay_time = Instrument.control(
+            "RDT",
+            "SDT %f",
+            """ Control delay time.""",
+            get_process=(lambda v: Racal1992.decode(v, "DT"))
+        )
+
+    special_function_enable = Instrument.setting(
+            "SF%s",
+            """ Enable or disable special function. True=enable, False=disable""",
+            values={True: "E", False: "D"},
+            map_values=True
+        )
+
+    # FIXME: not tested on real instrument!
+    special_function_number = Instrument.control(
+            "RSF",
+            "S%d",
+            """ Control special function.""",
+            get_process=(lambda v: Racal1992.decode(v, "SF"))
+        )
+
+    # FIXME: not tested on real instrument!
+    total_so_far = Instrument.measurement(
+            "RF",
+            """ Total number of events so far.""",
+            get_process=(lambda v: Racal1992.decode(v, "RF"))
+        )
+
+    software_version = Instrument.measurement(
+            "RMS",
+            "Instrument software version",
+            get_process=(lambda v: Racal1992.decode(v, "MS"))
+        )
+
+    gpib_software_version = Instrument.measurement(
+            "RGS",
+            "GPIB software version",
+            get_process=(lambda v: Racal1992.decode(v, "GS"))
+        )
+
+    device_type = Instrument.measurement(
+            "RUT",
+            """Unit device type. Should return 1992 for a Racal-Dana 1992
+            or 1991 for a Racal-Dana 1991.""",
+            get_process=(lambda v: Racal1992.decode(v, "UT"))
+        )
+
+    math_mode = Instrument.setting(
+            "M%s",
+            """ Enable or disable math mode. True=enable, False=disable""",
+            values={True: "E", False: "D"},
+            map_values=True
+        )
+
+    math_x = Instrument.control(
+            "RMX",
+            "SMX %f",
+            """ Control math constant X.""",
+            get_process=(lambda v: Racal1992.decode(v, "MX"))
+        )
+
+    math_z = Instrument.control(
+            "RMZ",
+            "SMZ %f",
+            """ Control math constant Z.""",
+            get_process=(lambda v: Racal1992.decode(v, "MZ"))
+        )
+
+    trigger_level_a = Instrument.control(
+            "RLA",
+            "SLA %f",
+            """ Control trigger level for channel A""",
+            get_process=(lambda v: Racal1992.decode(v, "LA"))
+        )
+
+    trigger_level_b = Instrument.control(
+            "RLB",
+            "SLB %f",
+            """ Control trigger level for channel B""",
+            get_process=(lambda v: Racal1992.decode(v, "LB"))
+        )
+
+    def __init__(self, adapter, name="Racal-Dana 1992", **kwargs):
+        kwargs.setdefault('write_termination', '\r\n')
+
+        super().__init__(
+            adapter,
+            name,
+            **kwargs
+        )
+
+    def read(self):
+        return self.read_bytes(21).decode('utf-8')
```

Comment:
`utf-8` decoding will fail for every byte value>=128.
Are you sure all the bytes returned are < 128?
If not `latin-1` decoding should do the job.

Reply:
Yes, I'm sure all bytes values are <128. It's not broken. Let's not fix it.